aiohttp/web_request.py

```diff
@@ -402,11 +400,6 @@ def keep_alive(self):
         """Is keepalive enabled by client?"""
         return not self._message.should_close
 
-    @property
-    def time_service(self):
```

Comment:
Wonder if that would be a breaking change.

Reply:
Did not understand what you mean.

Reply:
Should be fine. As I remember time_service is not documented

Reply:
@socketpair Removing public attribute/method/property may break someone's code who depends on it. 

@fafhrd91 Ah, cool then.